apple/testing/default_runner/BUILD

```
@@ -63,6 +63,7 @@ exports_files([
     "macos_test_runner.template.xctestrun",
     "tvos_test_runner.template.sh",
     "watchos_test_runner.template.sh",
+    "xctrunner_entitlements_template.template.plist",
```

This name is sorta funny compared to the rest, is that intentional?
```suggestion
    "xctrunner_entitlements.template.plist",
```